src/ext/browser-recorder.js

```diff
@@ -9,6 +9,8 @@ const DEBUG = false;
 
 const hasInfoBar = (self.chrome && self.chrome.braveWebrecorder != undefined);
 
+const IS_AGREGORE = navigator.userAgent.includes("AgregoreDesktop");
+
```

Comment:
For me, the `navigator.userAgent` is `'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) agregore-browser/1.3.4 Chrome/100.0.4896.160 Electron/18.3.4 Safari/537.36'` so this check would fail...

Reply:
That's weird. Is this in all frames?

Reply:
Using `agregore-browser` instead